### DISCUSSION TITLE
ET Pro Rules Oink Codes

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#9995] [Unsupported Versions] [answered]
Hello!

According to Proofpoint, I need seperate Oink Codes per sensor. How to I configure these different Oink Codes? The SO documentation describes only the general activation of the Pro Rules with one Oink Code [here](https://docs.securityonion.net/en/2.3/rules.html#et-pro-proofpoint). Thanks in advance!

Cheers, Ben

**Comment by InfosecGoon** [comment, marked as answer]
Proofpoint may require a separate license per sensor, but the oink code is only used on the Manager -- the Manager pulls down the ruleset, makes any tuning changes that you've configured using idstools, adds any local rules you've put in local.rules, and then uses Salt to push the final rule set out to the sensors.

**Reply by ben-sec** [reply]
Thanks!
